sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -596,6 +597,7 @@ def receive_message(
             return wrapped_message
         except HttpResponseError as error:
             process_storage_error(error)
+        return None
```

Comment:
Looks like you missed one :)

Reply:
Good catch! I was curious as to why this didn't show up in the Mypy output, but I can see why now as it's return type was Optional and so returning `None` was considered valid